_pages/algorithm-development/comparing-algorithms.md

```diff
@@ -43,6 +43,8 @@ Got all that? Great! Lets dive in.
     - Ensure that your evaluation algorithm is passing data to your evaluatable algorithms correctly! If they expect json, make sure you're passing something that can be serialized into json.
     - If some of your evaluatable algorithms handle input differently, make an an `if` statement that pattern matches the 'evaluated_algorithm' key. For more info check out the example above.
     - Make sure to publish your algorithm before using it.
+    
```

Comment:
Slightly pedantic but could we add "/" so IO becomes I/O?

Reply:
agree, will do in master

Reply:
I can make the change here, no sweat